rotkehlchen/tests/unit/accounting/evm/test_paraswap.py

```diff
@@ -114,8 +196,11 @@ def test_paraswap_swap_with_fee(accountant: 'Accountant'):
             price=Price((FVal(swap_amount_str) + FVal(fee_amount_str)) / FVal(receive_amount_str)),
             pnl=PNL(taxable=ZERO, free=ZERO),
             cost_basis=None,
-            index=2,
             extra_data=extra_data,
         )]
     expected_processed_events[0].count_cost_basis_pnl = True  # since it's not settable at ctor
+    expected_processed_events[1].count_cost_basis_pnl = True  # since it's not settable at ctor
+    expected_processed_events[2].count_cost_basis_pnl = True  # since it's not settable at ctor
     assert pot.processed_events == expected_processed_events
+    assert pot.pnls[AccountingEventType.TRANSACTION_EVENT].taxable == FVal(swap_amount_str) + ONE
```

Comment:
I believe is clearer if you put where they come from

```suggestion
    assert pot.pnls[AccountingEventType.TRANSACTION_EVENT].taxable == events[0].balance.amount + events[1].balance.amount
```

Then it is easier to adjust later and to understand it. Same below